packages/geoadmin-layers/src/parsers/externalWMSCapabilitiesParser.ts

```diff
@@ -392,60 +503,67 @@ export default class WMSCapabilitiesParser {
         }
 
         if (!layerExtent) {
-            const msg = `No layer extent found for ${layerId} in ${this.originUrl.toString()}`
+            const msg = `No layer extent found for ${layerId} in ${this.capabilities.originUrl.toString()}`
             log.error(msg, layer, parents)
         }
 
         return layerExtent
     }
 
-    _getLayerAttribution(layerId, layer) {
+    _getLayerAttribution(layerId: string, layer: CapabilityLayer) {
         let title = null
         let url = null
+
         try {
-            if (layer.Attribution || this.Capability.Layer.Attribution) {
-                const attribution = layer.Attribution || this.Capability.Layer.Attribution
+            if (layer.Attribution || this.capabilities.Capability?.Layer?.Attribution) {
+                const attribution =
+                    layer.Attribution || this.capabilities.Capability?.Layer?.Attribution
                 url = attribution.OnlineResource
                 title = attribution.Title || new URL(attribution.OnlineResource).hostname
             } else {
-                title = this.Service?.Title || new URL(this.Service?.OnlineResource).hostname
+                title =
+                    this.capabilities.Service?.Title ||
+                    new URL(this.capabilities.Service?.OnlineResource).hostname
             }
         } catch (error) {
+            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
Isn't it a bit weird to pass the error object to the string builder here?
Shouldn't we do something like `${error.message}` or something similar?
We could also clearly do a `${error.toString()}` (that's what TypeScript is complaining about here, isn't it? that we are passing non string param to the string concat)